index.html

```diff
@@ -3443,155 +3470,286 @@ <h2>Processing a Manifest</h2>
+							</li>
+
+							<li id="processing-checks-pub-date">
+								<p>(<a href="#publication-date"></a>) If <var>processed["datePublished"]</var> is set
+									and is not a valid date or date-time per&#160;[[!iso8601]], this is a <a>validation
+										error</a>.</p>
+							</li>
+
+							<li id="processing-checks-inlanguage">
+								<p>(<a href="#language-and-dir"></a>) If <var>processed["inLanguage"]</var> is set,
+									check that each item in <var>processed["inLanguage"]</var> is <a
+										href="https://tools.ietf.org/html/bcp47#section-2.2.9"
+									>well-formed</a>&#160;[[!bcp47]]. Each non well-formed item is a <a>validation
+										error</a>.</p>
+							</li>
+
+							<li id="processing-checks-progression-dir">
+								<p>(<a href="#reading-progression-direction"></a>) If
+										<var>processed["readingProgression"]</var> is set and is not one of the <a
 										href="#reading-progression-direction">required directional values</a>, this is a
-										<a>validation error</a>.</li>
-								<li>Otherwise, do nothing.</li>
-							</ul>
-						</li>
-
-						<li id="processing-checks-urls">
-							<p>(<a href="#resource-categorization-properties"></a>) For each <a
-									href="#resource-categorization-properties">resource categorization property</a>
-								<var>term</var> in <var>processed</var>, check whether each object <var>P</var> in
-									<var>processed[term]</var> has <var>P["url"]</var> set:</p>
-							<ul>
-								<li>
-									<p>If not, this is a <a>validation error</a>.</p>
-								</li>
-								<li>
-									<p>Otherwise, check whether <var>P["url"]</var> is a valid URL&#160;[[!url]]. If
-										not, this is a <a>validation error</a>.</p>
-								</li>
-							</ul>
-						</li>
-
-						<li id="processing-checks-length">
-							<p>(<a href="#LinkedResource"></a>) For each property <var>term</var> in
-									<var>processed</var> that expects type <a>LinkedResource</a>, if
-									<var>term["length"]</var> is set and is not a valid number this is a <a>validation
-									error</a>.</p>
-						</li>
-
-						<li id="processing-checks-extension">
-							<p>If a <a>profile</a> specifies data validation checks, those steps are executed at this
-								point.</p>
-						</li>
-					</ol>
-				</li>
-				<li id="processing-end">
-					<p>Return <var>processed</var>.</p>
-				</li>
-			</ol>
+										<a>validation error</a>.</p>
+								<p>If <var>processed["readingProgession"]</var> is not set, set it to the value
+										"<code>ltr</code>".</p>
+							</li>
+
+							<li id="processing-checks-extension">
+								<p>If a <a>profile</a> specifies data validation checks, those steps are executed at
+									this point.</p>
+							</li>
+						</ol>
+					</li>
+
+					<li id="processing-end">
+						<p>Return <var>processed</var>.</p>
+					</li>
+				</ol>
+			</section>
+
+			<section id="internal-rep-visualization">
+				<h3>Data Visualization</h3>
```

Comment:
I would think this section should be non-normative.

I am not sure about the title either. Just some alternative that comes to my mind:

- "Data structure overview"
- "Overview of the internal data structure"

"Visualization" triggers the association with graphics in my mind...

Reply:
yes, when I ask for input it's because I generally hate what I came up with... :)

What if to combine with @rdeltour's earlier response we call the appendix "Internal Representation Notations"?

That way we could present one or more notations for the structure, if we were so inclined. And we could also make clear that the appendix is just one way of visualizing the data using a known syntax, instead of giving the impression it is normative or the only way.

Reply:
I ended up with "Internal Representation Data Models". Might be odd to talk about representations and then only have one, though. Will have to see how this evolves.

Reply:
I can live with that; as far as I am concerned, it is a go.

Another bike shedding issue: In some sense, with the current setup, the WebIDL stuff is not more important than the TOC algorithm (or maybe even less), so if the TOC is an appendix not a bona fide section, then the WebIDL should be, too. But I leave that to you.

Reply:
> But I leave that to you.

You could have just said you forgot to make it an appendix when you moved it doofus... because, well, that's what I did. :)